packages/client/src/components/SendTokens/components/TrasactionDetails.js

```diff
@@ -17,41 +27,81 @@ const TransactionDetails = () => {
-            backgroundSize: "contain",
-            backgroundPosition: "center",
-            backgroundRepeat: "no-repeat",
-          }}
-        />
+      {assetType === ASSET_TYPES.NFT && !isEmpty(selectedNFT) && (
+        <div className="is-flex is-flex-direction-column is-align-items-start">
+          <div className="flex-1">
+            <div
+              style={{
+                cursor: "pointer",
+                height: "150px",
+                width: "150px",
+                backgroundImage: `url(${selectedNFT.thumbnail.url})`,
+                backgroundSize: "contain",
+                backgroundPosition: "center",
+                backgroundRepeat: "no-repeat",
+              }}
+            />
+          </div>
+          <span className="is-size-4">{`#${selectedNFT.tokenId}`}</span>
+          <strong>{selectedNFT.name}</strong>
+        </div>
       )}
       <div className="mt-5">
-        <div className="border-light-top is-flex is-justify-content-space-between py-5">
-          <span className="has-text-grey">Sending to</span>
-          <span>{shortenString(recipient)}</span>
+        <div className="border-light-top is-flex py-5">
+          <span className="has-text-grey flex-1">Sent From</span>
+          <div className="is-flex is-flex-direction-column flex-1">
+            <strong>Creature Treasury</strong>
+            <span>
+              {address}
+              <button
+                type="button"
+                className="pointer border-none has-background-white"
+                onClick={() => clipboard.copy(address)}
+              >
+                <Svg name="Copy" />
+              </button>
+            </span>
+          </div>
         </div>
-        <div className="border-light-top is-flex is-justify-content-space-between py-5">
-          <span className="has-text-grey">Network fee</span>
-          <span>{transactionFee} FLOW</span>
+        <div className="border-light-top is-flex py-5">
+          <span className="has-text-grey flex-1">Sent To</span>
+          <div className="is-flex is-flex-direction-column flex-1">
+            <strong>{recipientName}</strong>
+            <span>
+              {recipient}
+              <button
+                type="button"
+                className="pointer border-none has-background-white"
+                onClick={() => clipboard.copy(recipient)}
+              >
+                <Svg name="Copy" />
+              </button>
+            </span>
+          </div>
         </div>
-        <div className="border-light-top is-flex is-justify-content-space-between py-5">
-          <span>Total</span>
-          <span>{tokenAmount}</span>
+        <div className="border-light-top border-light-bottom is-flex is-justify-content-space-between py-5">
+          <span className="has-text-grey flex-1">Estimated Network Fee</span>
+          <span className="flex-1">
+            <strong>
+              {`${transactionFee} `}
```

Comment:
Fees should be updated to look similar to this
"Estimated Network Fee": ~0.0000011 FLOW

Reply:
Got it! I'll render a medium number of the range with the `~`